permissionsdispatcher-processor/src/main/java/permissions/dispatcher/processor/JavaFileBuilder.java

```diff
@@ -133,7 +147,7 @@ static JavaFile createJavaFile(RuntimePermissionsAnnotatedElement element) {
                 .addMethod(createConstructor())
                 .addMethods(createNeedsPermissionMethods(element))
                 .addMethods(createNeedsPermissionsMethods(element))
-                .addMethod(createOnRequestPermissionsResult(element.getClassName(), permissionsMethods))
+                .addMethod(createOnRequestPermissionsResult(element.getClassName(), permissionsMethods, element))
```

Comment:
Please change arguments to only 'element'. Because first and second are valued of element.